### PR TITLE
docs: index of examples

### PR DESCRIPTION
**Pull request by alexeagle**
This is mostly so that Angular devs who are pointed here by the deprecation are better able to find their way around

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+# Language tooling
+
+## LESS, Sass, Stylus
+
+See styles directory inside the [examples/app](https://github.com/bazelbuild/rules_nodejs/tree/master/examples/app/styles) example.
+
+## TypeScript
+
+Most of the examples show TypeScript usage. Also look in packages/typescript/test for lots of handling of edge cases.
```
Comment (alan-agius4) [inline docs/examples.md:78]: Probably should add a link here were it’s mentioned `packages/typescript/test`

File: docs/examples.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+## Webpack
+
+See [examples/react_webpack](https://github.com/bazelbuild/rules_nodejs/tree/master/examples/react_webpack)
+
+## Rollup
+
+The example at [examples/webapp](https://github.com/bazelbuild/rules_nodejs/tree/master/examples/webapp) uses Rollup, and produces an app with ES5 and ES2015 variants ("differential loading") that gives faster loading in modern browsers without dropping support for legacy ones.
+
+## Parcel
+
```
Comment (alan-agius4) [inline docs/examples.md:69]: Maybe add an issue here? Or say it’s WIP?

File: docs/examples.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+Example: [examples/angular_bazel_architect](https://github.com/bazelbuild/rules_nodejs/tree/master/examples/angular_bazel_architect)
+
+**Google**: This toolchain is what we originally advertised as "Angular, Bazel and CLI" (ABC). It is based on Google's internal toolchain for building Angular, and has good performance characteristics. However it is harder to migrate to, because it doesn't have good compatibility for existing applications.
+
+The example has its own guide: [examples/angular](https://github.com/bazelbuild/rules_nodejs/tree/master/examples/angular)
+
+**View Engine**: If you're stuck on the older Angular compiler/runtime before Ivy, called View Engine, then your options are more limited. We don't support Angular 9 + View Engine + Bazel.
+
+Example: [examples/angular_view_engine](https://github.com/bazelbuild/rules_nodejs/tree/master/examples/angular_view_engine)
```
Comment (alexeagle) [inline docs/examples.md:24]: this one should point to 1.x branch since we'll break it when we stop linking workspace root
